framework/include/samplers/Sampler.h

```diff
@@ -111,6 +159,12 @@ class Sampler : public MooseObject, public SetupInterface, public DistributionIn
 
   /// Initial random number seed
   const unsigned int & _seed;
+
+  /// Data offsets for computing location based on global index
+  std::vector<unsigned int> _offsets;
+
+  /// Total number of rows
+  unsigned int _total_rows = 0;
```

Comment:
Are you referring to MooseRandom with this statement? These parameters are unrelated to random number stuff, they are for indexing into the sampler data